pkg/edgeview/src/system.go

```diff
@@ -938,6 +967,217 @@ func dispAFile(f os.FileInfo) {
 	fmt.Printf("%s, %v, %d, %s\n", f.Mode().String(), f.ModTime(), f.Size(), f.Name())
 }
 
+func getTarFile(opt string) {
+	execCmd := strings.SplitN(opt, "tar/", 2)
+	if len(execCmd) != 2 {
+		fmt.Printf("tar needs a / and path input\n")
+		return
+	}
+	source := execCmd[1]
```

Comment:
It would be nice to use filepath.Clean function to avoid booking through checks with some odd path notation.

Reply:
good to know. will add that.